filter/filter.go

```diff
@@ -191,8 +205,8 @@ func (c *Client) GetDimensionBytes(ctx context.Context, userAuthToken, serviceAu
 }
 
 // GetDimensions will return the dimensions associated with the provided filter id as an array of Dimension structs
-func (c *Client) GetDimensions(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, filterID string) (dims []Dimension, err error) {
-	b, err := c.GetDimensionsBytes(ctx, userAuthToken, serviceAuthToken, collectionID, filterID)
+func (c *Client) GetDimensions(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, filterID string, q QueryParams) (dims Dimensions, err error) {
```

Comment:
Strictly speaking this should trigger a V2 release due to a breaking change. However being small, imposing a V2 change on all the other api clients would be disproportionately disruptive.

Reply:
Yes, I agree - the only affected service is `dp-frontend-filter-dataset-conroller`.